src/main/java/cart/data/MemberJdbcRepository.java

```diff
@@ -0,0 +1,54 @@
+package cart.data;
+
+import cart.data.entity.CartMember;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class MemberJdbcRepository implements MemberRepository {
+
+    private SimpleJdbcInsert simpleJdbcInsert;
+
+    public MemberJdbcRepository(DataSource dataSource) {
+        this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("CART_MEMBER");
+    }
+
+    @Override
+    public List<CartMember> getMembers() {
+        String sql = "select * from CART_MEMBER";
+        JdbcTemplate jdbcTemplate = simpleJdbcInsert.getJdbcTemplate();
+
+        return jdbcTemplate.query(sql, carMemberRowMapper());
+    }
+
+    @Override
+    public Optional<CartMember> getMember(String email, String password) {
+        JdbcTemplate jdbcTemplate = simpleJdbcInsert.getJdbcTemplate();
+        String sql = "select * " +
+                "from CART_MEMBER " +
+                "where member_email = ? and member_password = ?";
+        CartMember cartMember;
+        try {
+            cartMember = jdbcTemplate.queryForObject(sql, carMemberRowMapper(), email, password);
+        } catch (Exception e) {
+            return Optional.empty();
+        };
+        return Optional.ofNullable(cartMember);
+    }
+
+    private RowMapper<CartMember> carMemberRowMapper() {
+        return (rs, rowNum) -> {
```

Comment:
해당 메서드를 호출할 때마다 RowMapper 가 생성될것으로 보여요 :)
필드로 만들지 않으신 이유가 있을까요?

```suggestion
    private final RowMapper<CartMember> carMemberRowMapper = (rs, rowNum) -> {
```

Reply:
앗 처음에 알려주신 내용을 바로 이해하지 못하여서 따로 메서드로 분리하여 만들었었습니다.
알려주신 내용처럼 필드로 만들어 수정하였습니다.